cmd/podman/play/kube.go

```diff
@@ -84,7 +84,18 @@ func kube(cmd *cobra.Command, args []string) error {
 		fmt.Fprintf(os.Stderr, l)
 	}
 
-	fmt.Printf("Pod:\n%s\n", report.Pod)
+	switch len(report.Pods) {
+	case 0:
+		return nil
+	case 1:
+		fmt.Printf("Pod:\n")
+	default:
+		fmt.Printf("Pods:\n")
+	}
+	for _, pod := range report.Pods {
```

Comment:
is there any way to print the containers in the pods underneath their pod? Right now, this looks like it will do
```
Pods:
pod-1
pod-2
Containers:
container-1-1
container-2-1
container-1-2
container-2-2
```
where I think it'd be clearer if we did
```
Pods:
pod-1
Containers:
container-1-1
container-2-1

pod-2
Containers:
container-1-2
container-2-2
```

Reply:
Good idea, I'll make this change